ckan/lib/helpers.py

```diff
@@ -1176,6 +1176,20 @@ def render_datetime(datetime_, date_format=None, with_hours=False):
 
     # if date_format was supplied we use it
     if date_format:
+
+        # See http://bugs.python.org/issue1777412
+        if datetime_.year < 1900:
+            date_format = date_format.replace('%y', str(datetime_.year)[-2:])
+            date_format = date_format.replace('%Y', str(datetime_.year))
```

Comment:
@wardi Here's a regexp `r'(?<!%)%y'` that uses negative lookbehind, but this will work correctly only for `%y` and `%%y`, and not for `%%%y` and so on.